fe/fe-core/src/main/java/com/starrocks/replication/ReplicationJob.java

```diff
@@ -982,11 +982,13 @@ private void removeRunningTasks() {
     }
 
     private boolean isAllTaskFinished() {
+        LOG.info("Replication job state: {}, finished tasks: {}/{}, database id: {}, table id: {}, transaction id: {}",
+                state, finishedTasks.size(), taskNum, databaseId, tableId, transactionId);
```

Comment:
this info log can't determine the the following true or false result. will the runningTasks.size() is necessary as well in this log?